sample-input/cosine-source/cosine-source.py

```diff
@@ -0,0 +1,165 @@
+import openmoc
+from math import *
+
+###############################################################################
+#                          Main Simulation Parameters
+###############################################################################
+
+options = openmoc.options.Options()
+
+num_threads = options.getNumThreads()
+track_spacing = options.getTrackSpacing()
+num_azim = 4
```

Comment:
I would revert back to using the defaults in `Options`, esp. since they are 4 and 1000 for `num_azim` and `max_iters`. We should probably agree upon some standard for when (and when not) to parse command-line runtime options in our examples - I've simply tried to make all of them use `Option` for consistency.